webapp/src/manifest.js

```diff
@@ -1,4 +1,28 @@
 // This file is automatically generated. Do not modify it manually.
 
-export const id = 'com.mattermost.plugin-starter-template';
-export const version = '0.1.0';
+const manifest = JSON.parse(`
```

Comment:
```suggestion
export const manifest = JSON.parse(`
```

is one way to fix this

Reply:
The manifest was actually exported as the default, but the test wasn't updated. I've actually updated to export both the manifest and the old id and version just to make migration easier for anyone updating their template.